src/controller/ProjectManager.py

```diff
@@ -364,3 +372,8 @@ def import_raw_data(self, path: str):
         """
         raw_data = DataFrame(FileManager.import_(path))
         self.get_project().set_raw_data(raw_data, path)
+
+    @staticmethod
+    def open_user_manual():
+        path = os.path.join(Path(__file__).parent.parent, "user_manual.pdf")
+        subprocess.Popen([path], shell=True)
```

Comment:
Not sure if that's my problem, but the user manual does not open for me, I get the following error: 
/bin/sh: /Users/flo/Documents/Kit/discrete-choice/src/user_manual.pdf: Permission denied
just fyi

Reply:
I think this is OS dependent. It's working fine for me.